src/js/messaging/components/ComposeButton.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class ComposeButton extends React.Component {
+  render() {
+    return (
+      <button
+          onClick={this.props.onClick}
```

Comment:
Instead of passing an outside `onClick` handler, probably can just create a function in this component that pushes the route for the compose page into history. Like `browserHistory.push('/compose')` or something.